pkg/test/resourcefixture/testdata/basic/datacatalog/v1alpha1/datacatalogtag/datacatalogtag-minimal/dependencies.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright %!s(int=2025) Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: datacatalog.cnrm.cloud.google.com/v1alpha1
+kind: DataCatalogEntryGroup
+metadata:
+  name: datacatalogentrygroup-minimal-${uniqueId}
+spec:
+  projectRef:
+    external: ${projectId}
+  location: us-central1
+  resourceID: dceg_${uniqueId}
+  displayName: Initial Minimal Entry Group
+
+---
+
+apiVersion: datacatalog.cnrm.cloud.google.com/v1alpha1
+kind: DataCatalogEntry
+metadata:
+  name: datacatalogentry-minimal-${uniqueId}
+spec:
+  entryGroupRef:
+    # Replace with a valid reference to your DataCatalogEntryGroup.
+    # The EntryGroup must exist in the same project and location (us-central1).
+    external: projects/${projectId}/locations/us-central1/entryGroups/dceg_${uniqueId}
```

Comment:
Could you reference it by name? This will guarantee the dependent resource to be existent (or it'll error out faster).